.github/workflows/static.yml

```diff
@@ -0,0 +1,73 @@
+# Workflow to deploy main and staging content to GitHub Pages
+name: Deploy main and staging content to GitHub Pages
```

Comment:
At first glance, it seems a little strange to have the same action trigger on pushes to both `main` and `staging` and trigger a deployment of both the `main` and `staging` branches, regardless of which the branch was that was pushed to.

I think it's very clever to do it this way, and I don't think we should change it - but maybe just a little comment explaining what the action does a bit better would be helpful. This is really great, thank you!